scripts/utils/build.js

```diff
@@ -73,6 +73,9 @@ function webpackBuild(pkgName, pkgPath) {
 
   console.log('pkgPath:', `${pkgName}`);
   if (`${pkgName}` === 'widget-call-history' || `${pkgName}` === 'widget-number-pad' || `${pkgName}` === 'widget-speed-dial' || `${pkgName}` === 'webex-sign-in-page' || `${pkgName}` === 'widget-voice-mail') {
+    if(`${pkgName}` === 'widget-call-history' || `${pkgName}` === 'widget-number-pad' || `${pkgName}` === 'widget-voice-mail' || `${pkgName}` === 'widget-speed-dial'){
+      return;
+    }
```

Comment:
All typescript widgets are failling webpack build.

Reply:
Are we planning to fix these build error as well in the next PR

Reply:
These builds are failing in node-upgrade branch also. So I was thinking of creating JIRAs for these errors.